### DISCUSSION TITLE
'cmath' file not found error for AMD builds - RESOLVED - issue with gcc 11, install gcc 12

### DISCUSSION THREAD

**Discussion by Speedway1** [ggml-org/llama.cpp#6615] [General]
**Problem**
"make" command fails with:
   'cmath' file not found

e.g.:
     make LLAMA_HIPBLAS=1

**Solution**
Check the version of GCC that you're on. If < 12 then install the latest. 
Eg: 
   sudo apt install libstdc++-12-dev

**Comment by bobfromjapan** [comment]
I got same error in Ubuntu 22.04+b4706, but it can be fixed with installing `libstdc++-12-dev` by apt. See following,

https://github.com/ROCm/ROCm/issues/1843
